readme.md

```diff
@@ -0,0 +1,205 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <img src="assets/images/capstone.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>1st Capstone Project Of Microverse (Module One)</b></h3>
+
+</div>
+<br/>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Europe-Global-Summit\]  is a project that was given by Microverse. We had to redesign the similar project to this one, but we had to include personal content with or own data and photos.](#-europe-global-summit--is-a-project-that-was-given-by-microverse-we-had-to-redesign-the-similar-project-to-this-one-but-we-had-to-include-personal-content-with-or-own-data-and-photos)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [Loom Video ](#loom-video-)
+  - [:computer: Getting Started ](#computer-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+    - [Prerequisites](#prerequisites-1)
+    - [Setup](#setup-1)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Europe-Global-Summit] <a name="about-project"> is a project that was given by Microverse. We had to redesign the similar project to this one, but we had to include personal content with or own data and photos.</a>
```

Comment:
[Optional] I suggest that you remove the description of the project from the header so as to differentiate between the project title and the description.

![image](https://user-images.githubusercontent.com/20554583/217743156-016f8406-f62e-4454-a6d4-e6d0512d1e0f.png)